src/Joystick/Joystick.cc

```diff
@@ -1123,6 +1325,13 @@ void Joystick::_buildActionList(Vehicle* activeVehicle)
     _assignableButtonActions.append(new AssignableButtonAction(this, _buttonActionGimbalLeft,    true));
     _assignableButtonActions.append(new AssignableButtonAction(this, _buttonActionGimbalRight,   true));
     _assignableButtonActions.append(new AssignableButtonAction(this, _buttonActionGimbalCenter));
+    _assignableButtonActions.append(new AssignableButtonAction(this, _buttonActionEmergencyStop));
+
+    // TODO(bzd) take channel nos from config, especially max
+    for (int ch = 8; ch <= 16;ch++) {
```

Comment:
```suggestion
    for (int ch = 9; ch <= 16;ch++) {
```
Channels 1-8 are PWM MAIN